GameData/RP-1/Science/Experiments/Cherenkov.cfg

```diff
@@ -0,0 +1,134 @@
+	baseValue = 20
+	scienceCap = 20
+	dataScale = 5 //
+	requireAtmosphere = False
+	situationMask = 48
+	biomeMask = 0
+	celestialBodies = Earth
+
+	RESULTS
+	{
+		default = High energy particle data detected by the Cherenkov detector has been logged.
+	}
+
+	KERBALISM_EXPERIMENT
+	{
+		// sample mass in tons. if undefined or 0, the experiment produce a file
+		SampleMass = 0
+		BodyAllowed = HomeBody
+	}
+}
+
+//ROK
+// ============================================================================
+// Replacing stock experiments
+// ============================================================================
+@PART[*]:HAS[@MODULE[DMModuleScienceAnimateGeneric]:HAS[#experimentID[RP0Cherenkov]]]:FOR[RP-0-Kerbalism]
+{
+	!MODULE[DMModuleScienceAnimateGeneric]:HAS[#experimentID[RP0Cherenkov]] {}
+	MODULE
+	{
+		name = Experiment
+		experiment_id = RP0Cherenkov
+	}
+}
+
+// ============================================================================
+// Reconfigure experiments(avoiding a global nuke to keep people happy)
+// strongly recommendeded to not change anything in this section
+// Separate from above due to Configure Groups.
+// ============================================================================
+@PART[*]:HAS[@MODULE[Experiment]]:AFTER[RP-0-Kerbalism]
+{
+	@MODULE[Experiment]:HAS[#experiment_id[RP0Cherenkov]]
+	{
+		%ec_rate = 0.0005
+		%data_rate = 5 //2 b/s
+		@data_rate /= 7862400 //3 months
+		%requires = OrbitMinEccentricity:0.04
+		%resources = 
+		%experiment_desc = A Cherenkov detector used for the categorization of high-energy particles from solar wind and cosmic rays, particularly gamma rays from beyond the solar system.
+	}
+}
+
+//==========================================//
+//	General Science Module					//
+//==========================================//
+
+@PART[RP0Cherenkov]:FOR[RP-0]
+{	
+	MODULE
+	{
+		name = DMModuleScienceAnimateGeneric
+		experimentID = RP0Cherenkov
+		
+		animationName = needle
+		experimentAnimation = true
+		experimentWaitForAnimation = true
+		keepDeployedMode = 0
+		showEndEvent = false
+		showStartEvent = false
+		showToggleEvent = false
+		startEventGUIName = Open Sensors
+		endEventGUIName = Close Sensors
+		toggleEventGUIName = Toggle Sensors
+		evaDistance = 1.85
+		
+		collectActionName = Retrieve Radiation Data
+		resetActionName = Discard Radiation Data
+		reviewActionName = Review Radiation Data
+		dataIsCollectable = true
+		experimentActionName = Log Radiation
+		rerunnable = true
+		hideUIwhenUnavailable = true
+		useActionGroups = true
+		useStaging = false
+		xmitDataScalar = 1.0
+		usageReqMaskExternal = 4
+		
+		deployingMessage = The radiation sensors are now exposed.
+		waitForAnimationTime = -1
+		resourceExperiment = ElectricCharge
+		resourceExpCost = 0
+		experimentsLimit = 1
+		totalScienceLevel = 1.0
+		asteroidReports = false
+	}
+}
+
+
+//	COSMIC RAY LEVEL 1.5
+//	Cherenkov Detector
+//	Based on the Geiger-Muller Counters used on many spacecraft.

```

Comment:
Yeah I'm not seeing where it actually includes the previous level here? So the comment is wrong? (Per above it's good it doesn't include any cosmic ray levels)